db/schema.rb

```diff
@@ -187,6 +187,8 @@
     t.string   "call_for_papers_dates_updates_subject"
     t.text     "call_for_papers_schedule_public_template"
     t.text     "call_for_papers_dates_updates_template"
+    t.string   "comment_subject"
```

Comment:
@raluka have you executed the new migration locally? Because there're no changes in your schema file...